specutils/tests/test_region_extract.py

```diff
@@ -34,7 +34,7 @@ def test_region_simple(simulated_spectra):
 
     sub_spectrum_flux_expected = np.array(FLUX_ARRAY)
 
-    assert quantity_allclose(sub_spectrum.flux.value, sub_spectrum_flux_expected)
+    assert_quantity_allclose(sub_spectrum.flux.value, sub_spectrum_flux_expected)
```

Comment:
Thought I was going crazy here for a minute and there wasn't actually a diff, then I noticed the underscore 😆